dali/kernels/audio/mel_scale/mel_filter_bank_args.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_AUDIO_MEL_SCALE_MEL_FILTER_BANK_ARGS_H_
+#define DALI_KERNELS_AUDIO_MEL_SCALE_MEL_FILTER_BANK_ARGS_H_
+
+namespace dali {
+namespace kernels {
+namespace audio {
+
+enum class MelScaleType {
+  SLANEY,
```

Comment:
```suggestion
  Slaney,
```
...to add to our discussion ;)

Reply:
done